src/main/java/io/pinecone/clients/AsyncIndex.java

```diff
@@ -846,6 +848,178 @@ public ListenableFuture<DescribeIndexStatsResponse> describeIndexStats(Struct fi
         return asyncStub.describeIndexStats(describeIndexStatsRequest);
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p>Example
```

Comment:
"Example" text seem to be missing colons in this file.

Reply:
yes!